pkg/osquery/table/corefoundation_helpers_darwin.go

```diff
@@ -82,6 +81,6 @@ func goValueFromCFPlistRef(ref C.CFPropertyListRef) interface{} {
 	case C.CFStringGetTypeID():
 		return goString(C.CFStringRef(ref))
 	default:
-		panic(fmt.Sprintf("unknown CF type id %v", typeID))
+		return "Unknown"
```

Comment:
This seemed safe enough -- this function and the one that calls it appear to be wholly unused.

Reply:
`""` might be better? Sentinel values are hard. Or `__unknown__`

Reply:
"Unknown" is what we use directly above the switch (line 74) so I thought that was probably appropriate enough